packages/components/accordion/__tests__/accordion.test.tsx

```diff
@@ -121,9 +121,7 @@ describe("Accordion", () => {
 
     expect(button).toHaveAttribute("aria-expanded", "false");
 
-    await act(async () => {
-      await user.click(button);
-    });
+    await user.click(button);
```

Comment:
The warning no longer appears without wrapping `await user.click` in `act`.
This issue was caused by multiple versions of `@testing-library/dom` being present.
see: https://github.com/testing-library/user-event/discussions/906#discussioncomment-6165530